Classes/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
Hence marking this PR as blocked. (The base-distribution PR is not blocked, though.)